src/management-system-v2/components/process-list.tsx

```diff
@@ -103,6 +112,20 @@ const ProcessList: FC<ProcessListProps> = ({
     onSettled: refreshData,
   });
 
+  const deleteSelectedProcesses = useCallback(() => {
```

Comment:
you could move this function to processes and use it together with ConfirmationButton, then you could delete the process-delete components

Reply:
Used deleteSelectedProcesses function from processes.tsx and declared its type as "Dispatch<SetStateAction<Key[]>>" in process-list.tsx. However, this declaration shows the error "Expected 1 arguments, but got 0." Would there be a better type to declare it as?